docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -275,6 +275,42 @@ Style/For:
 
 In this example the `Exclude` would only include `bar.rb`.
 
+=== Extending a parent .rubocop.yml using extends
+
+The optional directive `extends` allows a child `.rubocop.yml` to specify
+that it changes a parent `.rubocop.yml` without affecting path-relativity rules.
+
+Given the following configs:
+
+[source,yaml]
+----
+# .rubocop.yml
+AllCops:
+  Exclude:
+    - 'generated/**/*.rb'
+
+Layout/LineLength:
+  Enabled: true
+  Include:
+    - 'child_directory/**/*.rb'
+
+Rails/HasAndBelongsToMany:
+  Enabled: false
+----
+
+[source,yaml]
+----
+# child_directory/.rubocop.yml
+extends: '../../.rubocop.yml'
```

Comment:
This should only go up one level, so `extends: '../.rubocop.yml'`

Reply:
Changed!